modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAPUnderwritingTrigger.testcase

```diff
@@ -0,0 +1,106 @@
+@component-name = "portal-solutions"
+definition {
+
+	property custom.properties = "feature.flag.LPS-165493=true";
+	property osgi.modules.includes = "site-initializer-raylife-ap";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property release.feature.flags.disable.DISABLE_PRIVATE_LAYOUTS = "true";
+	property testray.main.component.name = "Site Initializer Raylife AP";
+
+	var testSiteName = "Raylife AP";
+	var testSiteURL = "raylife-ap";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Raylife.createRaylifeSite(siteName = "Raylife D2C");
+
+		Raylife.createRaylifeSite(siteName = "Raylife AP");
+
+		User.logoutPG();
+
+		RaylifeAP.userLoginInRaylife(email = "scott.producer");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		User.logoutPG();
+
+		User.firstLoginPG();
+
+		Raylife.tearDownCP();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			User.logoutPG();
+		}
+	}
+
+	@description = "LPS-163191 - Verify if user can add a new Accident Citation."
+	@priority = 4
+	test CanAccidentCitationBeAdded {
+		property test.name.skip.portal.instance = "RaylifeAPUnderwritingTrigger#CanAccidentCitationBeAdded";
+
+		task ("Given that an user go to Raylife AP and create a new application") {
+			RaylifeAP.goToRaylifeUserProfileMenu(raylifeSite = "Raylife AP");
+
+			RaylifeApplicationModal.openNewApplicationFlow(
+				insurance = "Personal",
+				product = "Auto");
+		}
+
+		task ("When the user fullfil the creation forms until Vehicle Info") {
+			RaylifeApplicationModal.typeAllNewApplicationContactInfoFields(
+				apt = 0000,
+				city = "New York",
+				emailAddress = "test@liferay.com",
+				firstName = "Testing",
+				lastName = "Model",
+				phoneNumber = 0123456789,
+				state = "NY",
+				streetAddress = "Address",
+				zipCode = 12345);
+
+			RaylifeApplicationModal.typeAllNewApplicationVehicleInfoFields(
+				annualMileage = 50000,
+				make = "Volvo",
+				model = "v90 cross country",
+				ownership = "Own",
+				primaryUse = "Business",
+				year = 2019);
+		}
+
+		task ("And the user fullfil Driver Info and add Accident Citation") {
+			RaylifeApplicationModal.typeAllNewApplicationDriverInfoFields(
+				addAccidentCitation = "true",
+				ageFirstLicenced = 18,
+				firstName = "Testing",
+				gender = "Female",
+				highestEducation = "Masters",
+				lastName = "Model",
+				maritalStatus = "Single",
+				millitaryAffiliation = "None",
+				occupation = "Engineer",
+				relationToContact = "Spouse");
+
+			RaylifeApplicationModal.selectOptionInDropdownByName(
+				field = "accident-citation",
+				text = "Accident - Not at fault");
+
+			Button.click(button = "Next");
+		}
+
+		task ("Then check if the user could add an Accident Citation") {
```

Comment:
Please check the task block and final assertion. The task block is saying that the Citation can be added, but the assertion is checking that something is not there. 

Please add the following task blocks / steps:
1. Complete the application
2. Navigate to the application
3. Assert that the citation was added/visible.